KatyaBebeshko/HW5/task3.py

```diff
@@ -0,0 +1,21 @@
+# Task3. Print Fibonacci numbers up to the entered number n, using cycles.
+# (Sequence of Fibonacci numbers 0, 1, 1, 2, 3, 5, 8, 13, etc.)
+
```

Comment:
another way of implementation code:
n = int(input('enter some number from fibonacci list='))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')